plugins/package-managers/node/src/main/kotlin/Yarn.kt

```diff
@@ -100,7 +100,7 @@ class Yarn(
 }
 
 /**
- * Parse the given [output] of a Yarn _info_ command to a [PackageJson]. The output is typically a JSON object with the
+ * Parse the given [stdout] of a Yarn _info_ command to a [PackageJson]. The output is typically a JSON object with the
```

Comment:
Maybe say explicitly in the commit message "This function is not suitable for parsing the JSON objects on `stderr` as-is.".